.github/workflows/tests_and_release.yml

```diff
@@ -0,0 +1,68 @@
+name: Validate and Release connector
+on:
+  pull_request:
+    branches:
+      - master
+  release:
+    types: [published]
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Tableau Connector SDK
+        uses: actions/checkout@v4
+        with:
+          repository: tableau/connector-plugin-sdk
+          ref: tableau-2024.2
+
+      - name: Checkout CrateDB connector
+        uses: actions/checkout@v4
+        with:
+          path: cratedb-tableau-connector
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
+
+      - name: Set up Tableau connector packager
+        working-directory: connector-packager
+        run: |
+          python -m venv .venv
+          source ./.venv/bin/activate
+          python -m pip install setuptools
+          python setup.py install
+
+      - name: Validate connector
+        working-directory: connector-packager
+        run: |
+          echo Workflow: Validating connector
+          source ./.venv/bin/activate
+          python -m connector_packager.package --validate-only $GITHUB_WORKSPACE/cratedb-tableau-connector/cratedb_jdbc
+
+      - name: Build connector
+        if: ${{ github.event_name == 'release' }}
+        working-directory: connector-packager
+        id: build-connector
+        run: |
+          echo Workflow: Building connector connector 
+          source ./.venv/bin/activate
+          python -m connector_packager.package $GITHUB_WORKSPACE/cratedb-tableau-connector/cratedb_jdbc
```

Comment:
Unfortunately you cannot specify the output, so we depend on 'find' to figure out where the file is

Reply:
Thanks. 🤷